ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -136,8 +136,30 @@ module.exports = Marionette.LayoutView.extend({
         }));
     },
     focus: function () {
+        this.autoLoadDefaultIfSet();
         this.queryContent.currentView.focus();
     },
+    autoLoadDefaultIfSet: function () {
+        let userDefaultTemplate = user.getQuerySettings().get('template');
+        if (userDefaultTemplate) {
+            let sorts = userDefaultTemplate['querySettings'] && userDefaultTemplate['querySettings'].sorts;
+            if (sorts) {
+                sorts = sorts.map(sort => ({ attribute: sort.split(',')[0], direction: sort.split(',')[1] }));
+            }
+            this.model.set({
+                        type: 'custom',
+                        title: userDefaultTemplate['name'],
+                        filterTree: userDefaultTemplate['filterTemplate'],
+                        src: (userDefaultTemplate['querySettings'] && userDefaultTemplate['querySettings'].src) || '',
+                        federation: (userDefaultTemplate['querySettings'] && userDefaultTemplate['querySettings'].federation) || 'enterprise',
+                        sorts: sorts, 
+                        'detail-level': (userDefaultTemplate['querySettings'] && userDefaultTemplate['querySettings']['detail-level']) || 'allFields',
```

Comment:
✏️ should this be renamed as well?

Reply:
This is used in quite a few places in the code. I can do a refactor of it. Are you thinking result-view?